clusters/l3-sqnc/capacity/nginx/release.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: nginx
+  namespace: capacity
+spec:
+  install:
+    remediation:
+      retries: -1
+  releaseName: nginx-ingress-controller
+  chart:
+    spec:
+      chart: nginx-ingress-controller
+      sourceRef:
+        kind: HelmRepository
+        name: bitnami
+  interval: 10m0s
+  # Default values
```

Comment:
is this meant to stay in?

Reply:
Yea, Dave just confirmed (being in the office perks)